### PR TITLE
Tweak: Beta-Tester newsletter dialog trigger.

### PR DESCRIPTION
**Pull request by nuritsha**

### CODE REVIEW

File: assets/dev/js/admin/beta-tester/beta-tester.js [hunk 1]
```diff
@@ -1,35 +1,16 @@
...
 
 class BetaTesterModule extends elementorModules.ViewModule {
-	getDefaultSettings() {
-		return {
-			selectors: {
-				betaSelect: '.elementor_beta select',
-			},
-		};
-	}
+	getDefaultSettings() {}
```
Comment (kobizz) [inline assets/dev/js/admin/beta-tester/beta-tester.js]: Remove all the empty methods

File: core/admin/admin.php [hunk 2]
```diff
@@ -720,7 +720,12 @@ public function init_new_template() {
 	 * @access public
 	 */
 	public function init_beta_tester( $current_screen ) {
-		if ( 'elementor_page_elementor-tools' === $current_screen->id ) {
+		if ( 'yes' !== get_option( 'elementor_beta', 'no' ) ) {
+			return;
+		}
+
+		$beta_tester_user_meta = get_user_meta( get_current_user_id(), User::BETA_TESTER_KEY, true );
+		if ( empty( $beta_tester_user_meta ) && ( 'elementor_page_elementor-tools' === $current_screen->id ) ) {
```
Comment (kobizz) [inline core/admin/admin.php]: Just `!`

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/js/admin/beta-tester/beta-tester.js [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
+				beta_tester_first_to_know: '#beta-tester-first-to-know',
+			},
+		};
+	}
+
+	getDefaultElements() {
+		let elements = {};
+		const selectors = this.getSettings( 'selectors' );
+
+		elements.$beta_tester_first_to_know = jQuery( selectors.beta_tester_first_to_know );
```
Comment (kobizz) [inline assets/dev/js/admin/beta-tester/beta-tester.js]: Usually all our `js` variables are camelCase

File: assets/dev/js/admin/beta-tester/beta-tester.js (hunk 1 above)
Comment (nuritsha) [reply assets/dev/js/admin/beta-tester/beta-tester.js]: done

File: core/admin/admin.php (hunk 2 above)
Comment (nuritsha) [reply core/admin/admin.php]: done

File: assets/dev/js/admin/beta-tester/beta-tester.js (hunk 3 above)
Comment (nuritsha) [reply assets/dev/js/admin/beta-tester/beta-tester.js]: done

**Pull request merged**